src/colorizer/AnnotationData.ts

```diff
@@ -504,11 +504,14 @@ export class AnnotationData implements IAnnotationData {
         const labelData = annotationData.labelData[labelIdx];
         const isBoolean = labelData.options.type === LabelType.BOOLEAN;
         const name = labelNames[labelIdx];
-        const value = row[name]?.trim();
+        let value = row[name]?.trim();
         // Ignore invalid values (and omit boolean false values)
-        if (value === undefined || value === "" || (isBoolean && value === BOOLEAN_VALUE_FALSE)) {
+        if (value === undefined || value === "" || (isBoolean && value.toLowerCase() === BOOLEAN_VALUE_FALSE)) {
           continue;
         }
+        if (isBoolean) {
+          value = BOOLEAN_VALUE_TRUE;
+        }
```

Comment:
Fixes a bug I noticed where, if an annotation file had been opened and saved from Excel, it would no longer be recognized as a boolean label because Excel saves booleans as `TRUE/FALSE` instead of `true/false`.